### ISSUE TITLE
Trading money without any tests

### ISSUE THREAD

**Issue by minid33** [kelvinau/crypto-arbitrage#7]
Where are the unit/integration tests? Seems like a risky proposition to run a fiscal operation without Quality Assured code.

**Comment by kelvinau** [comment]
I agree that. That will be probably the next step I will work it on. For now you can just run it in a mock mode (by default) to see the result.
